bitnami/kafka/values.yaml

```diff
@@ -159,7 +159,7 @@ logRetentionHours: 168
 logSegmentBytes: _1073741824
 ## @param logsDirs A comma separated list of directories under which to store log files
```

Comment:
We may rephrase this param description or add a note below to avoid future confusion.

Reply:
I reuse the wording from kafka docs "The directories in which the log data is kept" and added a ref to it: 
``` 
## @param logsDirs A comma separated list of directories in which kafka's log data is kept
## ref: https://kafka.apache.org/documentation/#brokerconfigs_log.dirs
logsDirs: /bitnami/kafka/data
```